src/Configuration/src/SpringBootCore/SpringBootHostBuilderExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.CommandLine;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Linq;
+
+namespace Steeltoe.Extensions.Configuration.SpringBoot
+{
+    public static class SpringBootHostBuilderExtensions
+    {
+        /// <summary>
+        ///  Sets up the configuration provider in spring boot style '.' separated values in CommandLine or as `SPRING_APPLICATION_JSON` Environment variable
```

Comment:
```suggestion
        ///  Sets up the configuration provider in spring boot style '.' separated values in CommandLine or as SPRING_APPLICATION_JSON Environment variable
```

I think the backticks here add noise to the way it's rendered by intellisense